packages/vue-pdf-viewer/src/components/PDFViewer.vue

```diff
@@ -1,223 +1,173 @@
+            docOptions.url = props.source
         }
-    },
-    async mounted() {
-        try {
-            const eventBus = new pdfjsViewer.EventBus()
-            const downloadManager = new pdfjsViewer.DownloadManager()
-            this.pdfViewer = new pdfjsViewer.PDFViewer({
-                ...this.pdfjsViewerOptions,
-                container: this.$refs.viewerContainer,
-                eventBus,
-                downloadManager
-            })
-
-            const docOptions = {
-                cMapUrl: this.pdfjsCMapUrl,
-                cMapPacked: true
-            }
-
-            if (this.source instanceof Uint8Array) {
-                docOptions.data = new Uint8Array(this.source)
-            } else {
-                docOptions.url = this.source
-            }
 
-            const documentLoadingTask = pdfjsLib.getDocument(docOptions)
-            this.pdfDocument = await documentLoadingTask.promise
+        const documentLoadingTask = pdfjsLib.getDocument(docOptions)
+        state.value.pdfDocument = await documentLoadingTask.promise
 
-            const event = new Event('PDFViewer:documentLoaded')
-            window.dispatchEvent(event)
+        const event = new Event('PDFViewer:documentLoaded')
+        window.dispatchEvent(event)
 
-            eventBus.on('pagesloaded', () => {
-                this.pdfViewer.currentScaleValue = this.currentScale = this.defaultScale
+        eventBus.on('pagesloaded', () => {
+            toRaw(state.value.pdfViewer).currentScaleValue = state.value.currentScale = props.defaultScale
 
-                const event = new Event('PDFViewer:pagesLoaded')
-                window.dispatchEvent(event)
+            const event = new Event('PDFViewer:pagesLoaded')
+            window.dispatchEvent(event)
 
-                const scrollbarWidth = this.$refs.viewerContainer.offsetWidth - this.$refs.viewerContainer.clientWidth
-                this.$refs.viewerControls.style.width = `calc(100% - ${scrollbarWidth}px)`
-            })
+            const scrollbarWidth = viewerContainer.value.offsetWidth - viewerContainer.value.clientWidth
+            viewerControls.value.style.width = `calc(100% - ${scrollbarWidth}px)`
+        })
 
-            this.pdfViewer.setDocument(this.pdfDocument)
-        } catch (error) {
-            this.$emit('error', error)
-            throw error
-        }
+        toRaw(state.value.pdfViewer).setDocument(toRaw(state.value.pdfDocument))
+    } catch (error) {
+        emit('error', error)
+        throw error
     }
-}
+})
 </script>
+
+<template>
+    <div class="pdf-viewer">
+        <div class="viewer-container" ref="viewerContainer">
+            <div class="viewer"></div>
+        </div>
+        <div class="controls" ref="viewerControls">
+            <div class="pages">
+                <span class="current">{{ currentPage }}</span> {{ $t('pdfViewer.pageOf') }} <span class="total">{{ pageCount }}</span>
```

Comment:
You removed the i18n-wrapper, how does the `$t()` work now?

Reply:
By using the Composition API `$t` is injected by `app.config.globalProperties` and is therefore usable app wide because of the global scope. I think it's possible to change the scope but default seems to be global and I did not see a reason to change that. I tested it in the new qes-ident webapp with the i18n config you already implemented in eprescription and it works as expected.

Reply:
I'm not sure how long this will be supported, I wasn't really expecting that even still exsists when you switch the mode to composition api. I think the "new way" of doing this is to have less global properties and just use things like `useI18n()` to get the needed functions.

If I'm correct, it would actually be pretty simple, you can just use `userI18n()` like you would normally and because vue-i18n is a peer dependency, this should always be available when this library is used.

Reply:
Do you remember where you got this info from? I could not find any hints that this is deprecating or an "old way". It also seems to be default.

Reply:
Yea, I think using local scope is the new default. I don't really get the benefits of having a local scope, but as the local scope is inheriting the global scope anyway, it doesn't really matter.